chain/indexer/src/lib.rs

```diff
@@ -92,7 +95,11 @@ pub struct Indexer {
 impl Indexer {
     /// Initialize Indexer by configuring `nearcore`
     pub fn new(indexer_config: IndexerConfig) -> Self {
+        tracing::info!(target: INDEXER, "Load config from {:?}...", &indexer_config.home_dir);
```

Comment:
```suggestion
        tracing::info!(target: INDEXER, "Load config from {}...", indexer_config.home_dir.display());
```

Reply:
If we use `...` to signify the process here, then I suggest to use spans instead (https://docs.rs/tracing/0.1.26/tracing/#spans) and adjust our reporting infra to display spans in a useful way (they may be hidden by default)